SampleApps/TestApp/AppDelegate.swift

```diff
@@ -12,20 +12,26 @@
 
 import AEPAssurance
 import AEPCore
+import AEPEdge
+import AEPEdgeConsent
 import AEPEdgeIdentity
 import AEPServices
 import Compression
 import UIKit
 
+@UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
-    // TODO: Set up the Environment File ID from your Launch property for the preferred environment
-    private let LAUNCH_ENVIRONMENT_FILE_ID = ""
+    // TODO: Set up the preferred Environment File ID from your mobile property configured in Data Collection UI
+    private let ENVIRONMENT_FILE_ID = ""
```

Comment:
private let ENVIRONMENT_FILE_ID = "yourAppId"

To match Android.

Reply:
thanks Calise, we can update that in a future PR